apps/main/src/loan/components/PageCrvUsdStaking/Statistics/AdvancedDetails.tsx

```diff
@@ -0,0 +1,30 @@
+import { Card, CardHeader, Stack } from '@mui/material'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { t } from '@lingui/macro'
+import ActionInfo from '@ui-kit/shared/ui/ActionInfo'
+import { SCRVUSD_VAULT_ADDRESS } from './constants'
+import networks from '@/loan/networks'
+
+const { Spacing } = SizesAndSpaces
+
+const ethereumChainId = 1
```

Comment:
Should we move this to `apps/main/src/loan/constants.ts`?
```suggestion
const ETHEREUM_CHAIN_ID = 1
```